style.css

```diff
@@ -0,0 +1,959 @@
+    width: 100%;
+    background-color: rgb(60, 62, 66);
+    gap: 2%;
+  }
+
+  .programs img {
+    width: 20%;
+    object-fit: scale-down;
+    margin-left: 40%;
+    margin-top: 10%;
+  }
+
+  .sponsors img {
+    width: 150px;
+  }
+
+  #topnav img {
+    max-height: 18px;
+  }
+
+  #topnav a {
+    margin-right: 2%;
+  }
+
+  #topnavList {
+    margin: 0%;
+    padding: 1%;
+    list-style: none;
+    display: inline-flex;
+    justify-content: space-evenly;
+    gap: 15%;
+    padding-right: 10%;
+  }
+
+  #topnavList li {
+    font-size: 1.2rem;
+    color: rgb(255, 255, 255);
+  }
+
+  .social {
+    max-height: 20px;
+    max-width: 10rem;
+    color: rgb(255, 255, 255);
+  }
+
+  .dropDown {
+    display: flex;
+    background-color: #fff;
+  }
+
+  .mlogo1 {
+    display: block;
+    text-decoration: none;
+    font-size: 40px;
+    font-weight: 900;
+    color: #742820;
+    cursor: pointer;
+    margin-left: 80px;
+    align-self: center;
+  }
+
+  #menuOpen {
+    display: none;
+  }
+
+  #menuList {
+    display: flex;
+    flex-direction: row;
+    justify-content: end;
+    gap: 2%;
+    align-items: center;
+    list-style: none;
+    padding: 0%;
+    margin: 0%;
+    width: 100%;
+  }
+
+  #menuList a {
+    font-size: 1.2rem;
+    color: #272a31;
+  }
+
+  .campaignBtn {
+    justify-content: end;
+    padding: 1vw;
+    color: #ec5242;
+    background-color: #fff;
+    font-weight: 500;
+    font-size: 15px;
+    margin-left: 40px;
+    border: 5px solid #ec5242;
+    margin-top: 10px;
+    margin-bottom: 5px;
+    margin-right: 8vw;
+  }
+
+  #intro {
+    background-image: url(./Assets/desktopShapes.jpg);
+    padding-top: 10%;
+    padding-left: 15%;
+    padding-right: 25%;
+  }
+
+  #hello {
+    font-size: 4rem;
+  }
+
+  .deskhello {
+    display: none;
+  }
+
+  #autoFest {
+    font-size: 5rem;
+    line-height: 6rem;
+  }
+
+  #eventDetail {
+    font-size: 1.2rem;
+    width: 70%;
+  }
+
+  #eventDetail p {
+    line-height: 2rem;
+  }
+
+  #date {
+    font-size: 2.5rem;
+  }
+
+  #location {
+    font-size: 1.5rem;
+  }
+
+  #mainProgram {
+    font-size: 3rem;
+  }
+
+  #programContainer {
+    display: flex;
+    gap: 3px;
+    width: 90%;
+  }
+
+  .programs {
+    flex-direction: column;
+    width: 20%;
+    padding: 0;
+  }
+
+  .programIcon {
+    max-width: 50%;
+    max-height: 50%;
+  }
+
+  .programTitle {
+    font-size: 2rem;
+    text-align: center;
+    margin-top: 2%;
+    margin-bottom: 2%;
+  }
+
+  .programDetail {
+    font-size: 1rem;
+    text-align: center;
+    padding: 2% 5% 10% 5%;
+  }
+
+  #bookBtn {
+    padding: 2%;
+    font-size: 2rem;
+    text-decoration: underline;
+    background-color: unset;
+    border: none;
+    margin: 5% 0%;
+  }
+
+  #guestHead {
+    font-size: 3rem;
+    margin: 0;
+  }
+
+  #guestcontainer {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    grid-template-rows: auto;
+    padding: 0 5%;
+    gap: 5%;
+    margin: 2%;
+  }
```

Comment:
- [ ] To create a good user experience it's important that you make sure your design looks as close to the `original` design as much as possible. To do that please make sure that each of your speaker's images has the same image size and matches that of the original design. 
- [ ] Also, Kindly consider adjusting your `font` sizes as they seem to be a little bigger as compared to that of the `original` design. 
See below for more details:

<table>
<tr>
<th align=“middle”>Behance design</th>
<th align=“middle”>Your design</th>
</tr>
<tr>
<td>
<img width="959" alt="Screenshot 2023-07-06 at 19 27 49" src="https://github.com/Godedict/Auto-Conference/assets/107192076/43737bff-9fcf-4dd1-beee-09c4546e5782">
</td>
<td>
<img width="1439" alt="Screenshot 2023-07-07 at 00 49 19" src="https://github.com/Godedict/Auto-Conference/assets/107192076/61735805-d065-4818-8177-04d6b9d28f3b">
</td>

Reply:
Picture sizes adjusted

Reply:
- Optional
As requested by the previous reviewer, I would recommend you follow up on this comment. Your design still doesn't quite look like the one provided. The font size is way large, images are disproportionately large and don't look good. margin-top for the main section of the index page is way too much for liking. I would recommend you fix these issues, so you can use this project in your portfolio.